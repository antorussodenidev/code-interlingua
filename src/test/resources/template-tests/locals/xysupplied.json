{
  "X": 1,
  "Y": 42
}
