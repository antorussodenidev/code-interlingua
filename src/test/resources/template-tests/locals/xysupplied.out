package foo;

class C {
  int x = 1;

  public static final int F = 42;

  C() {
    this.x = 3;
  }
}
